test/validation/api/dma/dma.c

```diff
@@ -269,6 +269,44 @@ static void test_dma_debug(void)
 	CU_ASSERT(odp_dma_destroy(dma) == 0);
 }
 
+static void test_dma_same_name(const char *name)
+{
+	odp_dma_param_t dma_param;
+	odp_dma_t dma, dma_a, dma_b;
+
+	odp_dma_param_init(&dma_param);
+	dma_param.compl_mode_mask = ODP_DMA_COMPL_SYNC;
+	dma_a = odp_dma_create(name, &dma_param);
+
+	CU_ASSERT_FATAL(dma_a != ODP_DMA_INVALID);
+
+	dma = odp_dma_lookup(name);
+
+	CU_ASSERT(odp_dma_to_u64(dma) == odp_dma_to_u64(dma_a));
```

Comment:
When no name is given this test is a bit questionable. One could assume that `ODP_DMA_INVALID` is also a valid return value from `odp_dma_lookup()`.

Reply:
Lookup with `NULL` seems not to be a thing so dropped that support/validation. Added a "named" word in the spec text which hopefully gives a clue what you can and cannot find with the lookup function. V3.